package.json

```diff
@@ -30,21 +30,22 @@
   "dependencies": {
     "@bsull/augurs": "^0.9.0",
     "@emotion/css": "11.13.5",
-    "@grafana/data": "^12.0.0",
+    "@grafana/data": "^12.1.0",
```

Comment:
`@grafana` package upgrades are apparently required as per the Compatibility Check action - this is a coincidence and not particular to this PR, but we may as well do them here :)